content/en/platform/corda/4.5/enterprise/node/setup/corda-configuration-file.md

```diff
@@ -208,7 +208,6 @@ Here are the contents of the `reference.conf` file:
         },
         },
     "database" : {
-        "runMigration" : "true",
         "schema" : "dbo",
         "transactionIsolationLevel" : "READ_COMMITTED"
```

Comment:
@vkolomeyko, should we also remove "transactionIsolationLevel" : "READ_COMMITTED". In CE 4.6 onwards, only "schema" : "dbo" is included.

Reply:
TBH, this is not something I know much about.
@blsemo, @adelel1 - do you know who would be the best person to answer @michelemarsden question?

Reply:
As far as I am aware, the DB harmonization only landed in CE 4.6, so 4.5 and earlier versions still support `runMigration` and `transactionIsolationLevel` in the config.

Reply:
Thanks for clarifying. I shall close this PR without applying the changes.